lib/mackerel/invitation.rb

```diff
@@ -0,0 +1,27 @@
+module Mackerel
+  class Invitation
```

Comment:
Currently this does nothing and not used. Is it necessary?

Reply:
I deleted the class.